### PR TITLE
[5.1] Invalid class

### PR DESCRIPTION
**Pull request by Denitz**
### Summary of Changes

Use correct class names, prevent unknown class names, mostly in PHPDoc

### Testing Instructions

Apply patch.

### Actual result BEFORE applying this Pull Request

See undefined class IDE warnings.

### Expected result AFTER applying this Pull Request

No IDE warnings for the changed files.

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: plugins/system/guidedtours/src/Extension/GuidedTours.php [hunk 1]
```diff
@@ -186,7 +189,7 @@ private function getTour($tourId)
     /**
      * Return a tour and its steps or null if not found
      *
-     * @param   TourTable  $item  The tour to load
+     * @param   CMSObject  $item  The tour to load
```
Comment (wilsonge) [inline plugins/system/guidedtours/src/Extension/GuidedTours.php:192]:
```suggestion
     * @param   \stdClass  $item  The tour to load
```

I think from the code?

Comment (Denitz) [reply plugins/system/guidedtours/src/Extension/GuidedTours.php:192]: @wilsonge But the `$item` here is a result of `Joomla\Component\Guidedtours\Administrator\Model::getItem()` which returns `CMSObject`

Comment (wilsonge) [reply plugins/system/guidedtours/src/Extension/GuidedTours.php:192]: Oh i'm just blind looking at the return type instead of the param 🤦‍♂️

**Review by wilsonge — APPROVED** [review]

**Comment by HLeithner** [comment]
This pull request has been automatically rebased to 5.1-dev.

**Review by rdeutz — APPROVED** [review]

**Comment by wilsonge** [comment]
As this is just doc block changes I'm merging this with the 2 good code reviews

**Pull request merged**